### PR TITLE
Docs

### LINKED ISSUE #13: Add instructions to integrate with jest-puppeteer

### LINKED ISSUE #38: Add GH issue templates

### PR DESCRIPTION
**Pull request by BeroBurny**
Closes #13
Closes #38

### CODE REVIEW

File: docs/API.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+commands MetaMask to submit a transaction. For this to work MetaMask has to be in a transaction confirmation state (basically promting the user to submit/reject a transaction). You can (optionally) pass an object with `gas` and/or `gasLimit`, by default they are `20` and `50000` respectively.
+
+
+<a name="sign"></a>
+## `metamask.sign(): Promise<void>`  
+commands MetaMask to sign a message. For this to work MetaMask must be in a sign confirmation state.
+
+<a name="approve"></a>
+## `metamask.approve(): Promise<void>`  
+enables the app to connect to MetaMask account in privacy mode
```
Comment (mpetrunic) [inline docs/API.md:96]: I think this also works for approving new network being added

File: docs/API.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+# dAppeteer API
```
Comment (morrigan) [inline docs/API.md:1]: Can we have 1-2 sentences of short introduction what dappeteer is and for what it can be used?

File: docs/JEST.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
+# dAppeteer with jest
+
+As dAppeteer is Puppeteer with Metamask using it with jest is pretty similar to `jest-puppeteer`
```
Comment (morrigan) [inline docs/JEST.md:3]:
```suggestion
As dAppeteer is Puppeteer with Metamask. Using it with jest is pretty similar to `jest-puppeteer`.
```

File: docs/API.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  seed?: string;
+  password?: string;
+}
+```
+
+<a name="getMetamask"></a>
+## `dappeteer.getMetamaskWindow(browser: Browser, version?: string): Promise<Dappeteer>`
+
+<a name="methods"></a>
+# dAppeteer methods
```
Comment (morrigan) [inline docs/API.md:42]: Are all MM API functions automatically supported by dappeteer?

File: docs/API.md (hunk 1 above)
Comment (BeroBurny) [reply docs/API.md:96]: add issue to test this #42

File: docs/API.md (hunk 4 above)
Comment (BeroBurny) [reply docs/API.md:45]: Only methods implemented by dappeteer.

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**